access/src/main/java/edu/unc/lib/dl/ui/controller/SearchActionController.java

```diff
@@ -106,6 +106,22 @@ private String formatQueryText(HttpServletRequest request) {
         searchRequest.setRootPid(RepositoryPaths.getContentRootPid().getURI());
         searchRequest.setApplyCutoffs(false);
         SearchResultResponse resultResponse = queryLayer.performSearch(searchRequest);
+
+        SearchState searchState = searchRequest.getSearchState();
```

Comment:
was there a reason not to reuse `doSearch` from this same class here? It probably needs a bit of cleaning up since we don't need to populate jsp search results anymore, but it seems to do the same stuff.

Also, we don't need to retrieve the facets in all cases (such as the full record page), so it would be best not to issue the facet query unless it's needed. You might need to add a parameter to state whether facets should be retrieved or not

Reply:
Seems like we can probably clear out `search()` set of methods in this class, plus some of the private methods they use as well, since I don't think we are going to have any calls to them anymore.